app/controllers/metrics_controller.rb

```diff
@@ -2,4 +2,14 @@ class MetricsController < ApplicationController
 	action_auth_level :index, :instructor
 	def index
 	end
+
+private
```

Comment:
Because this API supposed exposed to the public (It is a public API, but with authentication, which should be handled by the action auth level)